Bug Reporting/ARKEmailBugReporter.m

```diff
@@ -92,10 +94,17 @@ - (instancetype)initWithEmailAddress:(NSString *)emailAddress logStore:(ARKLogSt
 
 - (void)composeBugReport;
 {
+    [self composeBugReportWithScreenshotLog:YES];
+}
+
+- (void)composeBugReportWithScreenshotLog:(BOOL)logScreenshot;
+{
     ARKCheckCondition(self.bugReportRecipientEmailAddress.length, , @"Attempting to compose a bug report without a recipient email address.");
     ARKCheckCondition(self.mutableLogStores.count > 0, , @"Attempting to compose a bug report without logs.");
     
-    if (!self.screenFlashView) {
+    self.logScreenshot = logScreenshot;
```

Comment:
Might be best semantically to name the ivar `attachScreenshotToNextBugReport`, and use the passed in variable `logScreenshot` in the `if` statement below. But I don't have strong opinions here.